blueprints/cloud-operations/uptime-monitoring-tool/source/net-mon-agent/main.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
```

Comment:
can you format using yapf with 2 spaces for indent, and the following setup

`--style {based_on_style: google, indent_width: 2, SPLIT_BEFORE_NAMED_ASSIGNS: false}`

Reply:
Should be fine now!